portfolio/src/main/webapp/script.js

```diff
@@ -48,3 +48,11 @@ function showPics() {
     img.classList.add("image-hide");
   }
 }
+
+/* Async function that gets quote from servlet page and renders it to home page of portfolio. */
+async function getHelloUsingAsyncAwait() {
```

Comment:
The function name doesn't need to include the "UsingAsyncAwait", especially since that could change in the future.  I would suggest picking a name that describes _**what**_ the function does, not **_how_** it does it.  Since you're fetching data from /data, 'getData' seems more appropriate in this case.